implementations/rust/ockam/ockam_node/src/worker_builder.rs

```diff
@@ -40,6 +41,7 @@ where
             outgoing_ac: Arc::new(AllowAll),
             worker: self.worker,
             address: address.into(),
+            metadata: None,
```

Comment:
Wouldn't everything become much simpler if `metadata` was part of the mailbox?

Reply:
Yes, but it would be a redundant and possibly outdated information since it's stored within a shared structure in the router.
I'm more leaned toward keeping the builder a bit more complex and avoiding duplicates that could stray the casual contributor